src/main/java/com/fewlaps/quitnowcache/QNCache.java

```diff
@@ -195,4 +198,29 @@ private String getEffectiveKey(String key) {
         }
         return key;
     }
+    
+    /**
+     * Find 
+     */
+    List<String> findStartsWith(String someCuquiWord) {
+    	final String key = getEffectiveKey(someCuquiWord);
+
+    	return Collections.list(cache.keys())
+    			.stream()
+    			.filter(s -> s.startsWith(key))
+    			.collect(Collectors.toList());
+    }
+    
+    /**
+     * Finding coincidences
+     */
+    List<String> findStartsWithIfAlive(String someCuquiWord) {
```

Comment:
As the method to get elements from the cache is called "get", what do you think about naming this method `getItemsStartingWith`? Btw, probably there are people that doesn't know what `cuqui` means :·D Btw2, why not placing this method near the get one? It will be easy to get that this is the public API. Btw3: Why not making this method public? Is the only way to share it to the consumers.